docs/architecture-and-technical-specifications.md

```diff
@@ -262,6 +264,20 @@ age verification, this initiative will empower users, online services, and
 regulators to uphold digital safety and inclusivity while ensuring compliance 
 with Union law.
 
+It must however be understood that bypassing regional requirements is trivial 
+to most users with a simple online search, and the realities of the simplicity
+and the dangers of driving users to that SHALL also be taken seriously and the
+solution SHALL thus only be proportional to the real needs and real risks.
+
+The solution SHALL ensure that it will not be possible, regardless of future 
+malicious intent or being compromised, for any party to collect information of 
+who is using what services from the age verification flows, i.e. the parties 
+requesting age verification SHALL NOT learn the identity of the users from the 
+flow, and the parties providing verification SHALL NOT learn the identity of 
+the requesting parties, and SHOULD NOT learn the identity of the users if it 
+is feasible to avoid that.
```

Comment:
I think something similar to: "Additionally it MUST NOT be possible to uniquely fingerprint users across services or sessions based upon the age verification method."

This would preclude a UUID representing a user from being embedded in whatever verification method sent to services.  It would also even preclude an encrypted UUID that could only be decrypted by law enforcement for example.

Reply:
(I see this is already discussed later, but I think it is important to list requirements explicitly together in this regard)

Reply:
There should be indeed some additions for at least

- Ensuring users cannot be identified between services as being the same person
- Since the document recommends e.g. banks set up both age verification and identity verification, that it is impossible to create any logs or similar that could be used in combination to identify who identified when for which site's age verification request, maliciously or otherwise.